Shaders/PostProcess/ToneMapping/public/ToneMapping.fxh

```diff
@@ -17,6 +18,73 @@ float3 Uncharted2Tonemap(float3 x)
     return ((x*(A*x+C*B)+D*E)/(x*(A*x+B)+D*F))-E/F; // E/F = Toe Angle
 }
 
+float3 AgXDefaultContrastApprox(float3 x) 
+{
+    float3 x2 = x * x;
+    float3 x4 = x2 * x2;
+  
+    return + 15.5     * x4 * x2
+           - 40.14    * x4 * x
+           + 31.96    * x4
+           - 6.868    * x2 * x
+           + 0.4298   * x2
+           + 0.1191   * x
+           - 0.00232;
+}
+
+float3 AgX(float3 Color) 
+{
+    const float3x3 AgXTransform = float3x3(
+        0.842479062253094,  0.0423282422610123, 0.0423756549057051,
+        0.0784335999999992, 0.878468636469772,  0.0784336,
+        0.0792237451477643, 0.0791661274605434, 0.879142973793104);
```

Comment:
This will work incorrectly in OpenGL since OpenGL matrix are column-major in GLSL